src/pages/messages/components/NviStatusPage.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Box,
+  Paper,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography,
+} from '@mui/material';
+import { visuallyHidden } from '@mui/utils';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+import { useFetchNviCandidates } from '../../../api/hooks/useFetchNviCandidates';
+import { useFetchOrganization } from '../../../api/hooks/useFetchOrganization';
+import { BackgroundDiv } from '../../../components/styled/Wrappers';
+import { RootState } from '../../../redux/store';
+import { OrganizationApprovalStatusDetail } from '../../../types/nvi.types';
+import { isValidUrl } from '../../../utils/general-helpers';
+import { NviStatusTableRow } from './NviStatusTableRow';
+
+interface NviStatusPageProps {
+  activeYear: number | string;
+}
+
+export const NviStatusPage = ({ activeYear }: NviStatusPageProps) => {
+  const { t } = useTranslation();
+  const user = useSelector((store: RootState) => store.user);
+
+  const organizationQuery = useFetchOrganization(user?.topOrgCristinId ?? '');
+  const institution = organizationQuery.data;
+
+  const nviQuery = useFetchNviCandidates({ size: 1, aggregation: 'all', year: activeYear });
+  const aggregationKeys = Object.keys(nviQuery.data?.aggregations?.organizationApprovalStatuses ?? {});
+  const aggregationKey = aggregationKeys.find((key) => isValidUrl(key));
+  const nviAggregations = nviQuery.data?.aggregations?.organizationApprovalStatuses[aggregationKey ?? ''] as
+    | OrganizationApprovalStatusDetail
+    | undefined;
+
+  return (
+    <BackgroundDiv>
+      <Typography variant="h1" gutterBottom>
+        {t('tasks.nvi.institution_nvi_status')}
+      </Typography>
+
+      <TableContainer component={Paper} variant="outlined">
```

Comment:
TODO: Fiks feil for mobilvisning her!

Reply:
Løst ved å sette maks bredde for container: https://github.com/BIBSYSDEV/NVA-Frontend/pull/5707/commits/6860f5291641199a85c300daf1b8d45f325ff7e4